draft-pda-protocol.md

```diff
@@ -146,13 +146,164 @@ document.
 
 [OPEN ISSUE: Rework this section in light of issue #44.]
 
-The protocol is executed by a large set of clients and a small set of servers.
-We call the servers the *aggregators*. Each client's input to the protocol is a
-set of measurements (e.g., counts of some user behavior). Given the input set
-of measurements `x_1, ..., x_n` held by `n` users, the goal of a
-*private data aggregation (PPM) protocol* is to compute `y = F(x_1, ..., x_n)` for
-some aggregation function `F` while revealing nothing else about the
-measurements.
+The protocol is executed by a large set of clients and a small set of
+servers.  We call the servers the *aggregators*. Each client's input
+to the protocol is a set of measurements (e.g., counts of some user
+behavior). Given the input set of measurements `x_1, ..., x_n` held by
+`n` users, the goal of a *privacy preserving measurement (PPM)
+protocol* is to compute `y = F(x_1, ..., x_n)` for some aggregation
+function `F` while revealing nothing else about the measurements.
+
+This protocol is extensible and allows for the addition of new
+cryptographic schemes that compute new functions. The current
+version supports two schemes:
+
+* Prio {{CB17}}, which allows for aggregate statistics such as
+  sum, mean, histograms, etc. over a single value.
+
+* Heavy Hitters {{BBCp21}}, which allows for finding the most
+  common strings among a collection of clients (e.g., the
+  URL of their home page) as well as counting the number of
+  clients that hold a given string.
+
+This protocol is designed to work with schemes that use secret
+sharing. Rather than send its input in the clear, each client splits
+its measurements into a sequence of *shares* and sends a share to each
+of the aggregators. This provides two important properties:
+
+* It's impossible to deduce the measurement without knowing *all* of the shares.
+
+* It allows the aggregators to compute the final output by first
+   aggregating up their measurements shares locally, then combining
+   the results to obtain the final output.
+
```

Comment:
```suggestion
Aggregators also have the option of validating shares, as discussed in {{validating-inputs}}.
```

Reply:
The option? Isn't it required effectively?

Reply:
There were some deployments that wanted to use this without actually verifying any proofs 🤷 , relying on client attestation and authentication of some form to ensure the data is trusted.